packages/complaints/lib/pages/registration/complaints_details.dart

```diff
@@ -53,18 +58,21 @@ class _ComplaintsDetailsPageState
         form: () => bloc.state.map(
           create: (value) => buildForm(value),
           persisted: (value) =>
-              throw const InvalidComplaintsRegistrationStateException(),
+
+          throw const InvalidComplaintsRegistrationStateException(),
           view: (value) => buildForm(value),
         ),
         builder: (_, form, __) => BlocListener<BoundaryBloc, BoundaryState>(
           listener: (context, state) {
-            context.navigateTo(HomeRoute());
+            context.navigateTo(ComplaintsInboxWrapperRoute());
```

Comment:
Why is this changed to ComplaintsInboxWrapperRoute? HomeRoute is not available in the package so for this it should be popUntilRoot.